Sources/XcodeExport/XcodeTypographyExporter.swift

```diff
@@ -26,7 +26,7 @@ final public class XcodeTypographyExporter: XcodeExporterBase {
     public func export(textStyles: [TextStyle]) throws -> [FileContents] {
         var files: [FileContents] = []
 
-        // UIKit & SwiftUI UIFont extension
+        // UIKit UIFont extension
```

Comment:
Do we not need UIFont anymore when only using SwiftUI? I did mention it here for clarity since we needed it to calculate the lineSpacing.

Reply:
Oh I see what you mean here, I thought we simply wanted to note what extension file we are creating. I will add additional comments here to clarify!

Reply:
Also maybe the parameter `fontSwift`, i.e. UIFont extension, should be non-optional, since it'll be required for not only UIKit but also SwiftUI?